### PR TITLE
Created Content Editor: Form

### PR DESCRIPTION
**Pull request by kellyhp**
added rich text editor along with input for title and date

### CODE REVIEW

**Comment by kellyhp** [comment]
Failed typescript check because I added a new package called react-quill for the rich text editor. Included the css file that React Quill provides and edited the content to match the style of the page. There's probably a way I could make this more organized and simple, but idk.

File: app/(pages)/content/[collection]/edit/_components/Description/TextEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+'use client';
+import styles from './DescriptionForm.module.scss';
+import React, { useState } from 'react';
+import ReactQuill from 'react-quill';
+import './quill.snow.css';
```
Comment (Austin2Shih) [inline app/(pages)/content/[collection]/edit/_components/Description/TextEditor.tsx:5]: Does next.js complain about this? I'm not sure if we can just import css into components. We could put it as another "global" css file and have this get imported into the root the same way globals.scss is done.

**Review by Austin2Shih — CHANGES_REQUESTED** [review]
For the quill.css stuff, probably import that into the root the same way globals.scss is imported. I'm not sure if Next.js allows us to be using non-module css files like this. Also, in the future please make some sort of demo page so I can easily see the changes. Maybe you did tho and i could just be blind. In that case you're good lol

File: app/(pages)/content/[collection]/edit/_components/Description/TextEditor.tsx (hunk 1 above)
Comment (kellyhp) [reply app/(pages)/content/[collection]/edit/_components/Description/TextEditor.tsx]: I don't think next.js had any issues with it being css actually, but I changed it to scss for our peace of mind along with moving it into the globals folder. I also added a new page under /content route, so you can check out how it looks like.

**Review by Austin2Shih — APPROVED** [review]
Looks really good! Go ahead and merge it whenever 👍

**Pull request merged**